lib/chat_api/conversations.ex

```diff
@@ -214,6 +214,8 @@ defmodule ChatApi.Conversations do
   @spec archive_conversation(Conversation.t() | binary()) ::
           {:error, Ecto.Changeset.t()} | {:ok, Conversation.t()}
   def archive_conversation(%Conversation{} = conversation) do
+    Helpers.send_conversation_state_update(conversation, %{"status" => "archived"})
+
     update_conversation(conversation, %{archived_at: DateTime.utc_now()})
```

Comment:
Is `archive_conversation` used anywhere in the app besides testing? I see that we use `archive_conversations` in an Oban worker but not the singular version of the method. If it's just for testing purposes, I say we keep it here in order to manually make the slack message appear. This was the only way I was able to reproduce the screenshot above.

Reply:
hmm yeah i don't think this method is used anywhere anymore... happy to keep this here for testing in the meantime